ldraw_to_scad

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+""" Translate LDraw library or file to OpenSCAD library or file. """
+
+import os
+import argparse
+from ldraw_to_scad import LDrawConverter
+
+
+def translatedir(converter, src, dest, selfcontained=False):
```

Comment:
It might be nice to name these things camel_case, both the function and it's parameters.

Reply:
Not really new code but just moved around. But yes, I could change that as suggested.